azure/notebooks/azure_mnmg_xgboost.ipynb

```diff
@@ -0,0 +1,1323 @@
+{
```

Comment:
<p>Can we can provide this as a separate file in a <code>config</code>  folder instead of writing it here? And mention that there's a file provided. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai/cloud-ml-examples/pull/127/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>

Reply:
<p>I guess that is better. </p>